members.php

```diff
@@ -421,15 +420,15 @@ function members() {
 		else {
             $query_custom = " WHERE `staff_level` <= " . User::STAFF_HEAD_MODERATOR .
                 " ORDER BY `exp` DESC, `pvp_wins` DESC";
-			$list_name = 'Top 10 Users - Highest Exp';
+			$list_name = 'Top 15 Users - Highest Exp';
 			$view = 'highest_exp';
 
 		}
 
 		// Pagination
 		$users_per_page = 15;
 		$min = 0;
-		if(isset($_GET['min']) && $view != 'highest_exp' && $view != 'highest_pvp') {
+		if(isset($_GET['min']) && $view != 'highest_exp' && $view != 'highest_teams' && $view != 'highest_pvp') {

```

Comment:
At this point, what is the $view this is used
```suggestion
		if(isset($_GET['min']) && $view == 'online_users') {

```